web/styles/compose.css

```diff
@@ -1001,41 +1001,80 @@ textarea.new_message_textarea {
 }
 
 .enter_sends_choices {
+    display: flex;
+    flex-direction: column;
+    color: var(--color-text-popover-menu);
+
     .enter_sends_choice {
         display: flex;
         gap: 8px;
-        padding-top: 4px;
+        padding: 4px 10px;
+        align-items: flex-start;
+
+        &:hover {
+            background: var(--color-background-hover-popover-menu);
+            outline: none;
+        }
+
+        &:active {
+            background: var(--color-background-active-popover-menu);
+        }
 
         & input[type="radio"] {
```

Comment:
This selector should probably just be replaced with `.enter-sends-choice-radio` or something? Generally these element-based selectors are a bad pattern.

Reply:
Makes sense! Replaced `input[type="radio"]` selector with `.enter_sends_choice_radio`.